src/main/resources/messages/messages_zhhk.yml

```
@@ -121,19 +121,32 @@ captcha:
     usage_captcha: '&8[&6用戶系統&8] &f用法：《 /captcha %captcha_code 》'
     wrong_captcha: '&8[&6用戶系統&8] &c你所輸入的驗證碼無效，請使用 《 /captcha %captcha_code 》 再次輸入。'
     valid_captcha: '&8[&6用戶系統&8] &c你所輸入的驗證碼無效 ！'
-    # TODO captcha_for_registration: 'To register you have to solve a captcha first, please use the command: /captcha %captcha_code'
-    # TODO register_captcha_valid: '&2Valid captcha! You may now register with /register'
+    captcha_for_registration: '&8[&6用戶系統&8] &2要完成玩家註冊程序，請先執行驗証指令：《 /captcha %captcha_code 》'
+    register_captcha_valid: '&8[&6用戶系統&8] &2驗證成功！你現在可以使用指令《 /register 》進行註冊了～'
 
 # Verification code
 verification:
     # TODO code_required: '&3This command is sensitive and requires an email verification! Check your inbox and follow the email''s instructions.'
-    # TODO command_usage: '&cUsage: /verification <code>'
+    command_usage: '&8[&6用戶系統&8] &f用法：《 /verification <code> 》'
     # TODO incorrect_code: '&cIncorrect code, please type "/verification <code>" into the chat, using the code you received by email'
     # TODO success: '&2Your identity has been verified! You can now execute all commands within the current session!'
     # TODO already_verified: '&2You can already execute every sensitive command within the current session!'
     # TODO code_expired: '&3Your code has expired! Execute another sensitive command to get a new code!'
     # TODO email_needed: '&3To verify your identity you need to link an email address with your account!!'
 
+# Two-factor authentication
+two_factor:
+    code_created: '&8[&6用戶系統 - 兩步驗證碼&8] &b你的登入金鑰為&9「%c%code&9」&b，掃描連結為：%nl%&c %url'
+    # TODO confirmation_required: 'Please confirm your code with /2fa confirm <code>'
+    # TODO code_required: 'Please submit your two-factor authentication code with /2fa code <code>'
+    # TODO already_enabled: 'Two-factor authentication is already enabled for your account!'
+    # TODO enable_error_no_code: 'No 2fa key has been generated for you or it has expired. Please run /2fa add'
+    # TODO enable_success: 'Successfully enabled two-factor authentication for your account'
+    # TODO enable_error_wrong_code: 'Wrong code or code has expired. Please run /2fa add'
+    # TODO not_enabled_error: 'Two-factor authentication is not enabled for your account. Run /2fa add'
+    removed_success: '&8[&6用戶系統 - 兩步驗證碼&8] &2已成功關閉兩步驗證功能；&c為保障帳戶安全，請儘快於可行的情況下重新啟用本功能。'
```

Strings translated according to the original sentence
```suggestion
    removed_success: '&8[&6用戶系統 - 兩步驗證碼&8] &2已成功關閉兩步驗證功能。'
```
---
Due to some local infosec issues(people are unlikely to know how to secure things), the finalized translated string has added an extra (the part starts with `&c) warning, to notify the user re-enable 2FA feature as soon as possible, to protect their account security.

Example: https://www.scmp.com/business/banking-finance/article/2170183/hong-kong-halts-e-wallets-top-ups-after-users-reports